### PR TITLE
Mount romfs ourselves

### PR DESCRIPTION
**Pull request by Genwald**
- Runs plugins earlier.
- Avoids a potential crash if the rom is not yet mounted after >10 seconds.
- Avoids a potential crash if the game doesn't mount rom as "rom" and mounts too early for skyline to hook it.

Tested with smash ultimate.
Potential memory leak if the game mounts as "rom" as well? Wasn't sure if it'd be safe to free the game's buffer.

### CODE REVIEW

File: source/main.cpp [hunk 1]
```diff
@@ -7,6 +7,8 @@
 char ALIGNA(0x1000) exception_handler_stack[0x4000];
 nn::os::UserExceptionInfo exception_info;
 
+const char* RomMountName = "rom";
```
Comment (3096) [inline source/main.cpp:10]: not all games use "rom" tho?

**Comment by jam1garner** [comment]
Does this break games which have different mount names?

**Comment by jakibaki** [comment]
Pretty sure it does just that.

**Comment by Genwald** [comment]
In the case of a game using a different mount name, romfs will just be mounted as "rom" as well as whatever name the game chooses. I did not test with other games, but did test using a mount name other than rom on smash ultimate.

**Pull request merged**